recipes/kosudoku/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "kosudoku" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version | replace('-', '_') }}"
+
+source:
+  url: https://github.com/buzbarstow/kosudoku/archive/refs/heads/master.zip
```

Comment:
please use the commit hash of the repo. For example this one https://github.com/buzbarstow/kosudoku/archive/20bc89b5c017eecca3a07369b9b72f171a165162.zip

Reply:
I set a tag v1.0 from my fork. I guess it will simplify communications with collaborators.